deploy/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "bucket_name" {
+  default = "answerking-bucket-react"
+}
+
+variable "ssl_certificate_arn" {
+  description = "The certificate ARN for the provided domain. Be aware that for cloud front the certicate needs to be available in us-east-1."
+  type        = string
+  default     = ""
```

Comment:
is this meant to be empty?

Reply:
Wasn't sure about this. Thought that if someone wanted to re-create the distribution, it'd be best if they had the choice between choosing the ssl certificate from currect aws distribution or create their own. 

Also wasn't sure how to store it.